src-v2/Not/Not.Blazor/TM/NotChildren.razor

```diff
@@ -0,0 +1,62 @@
+﻿@using Not.Blazor.Components
+@using Not.Blazor.Forms
+@using Not.Blazor.Mud.Components
+@using Not.Blazor.Navigation
+@using Not.Blazor.TM.Forms.Components
+@using Not.Services
+
+@typeparam T where T : DomainEntity
+@typeparam TForm where TForm : NotForm<T>
+
+<MudPaper Elevation="0" Class="children-component half-width">
+@if (_parentBehind.Children.Any())
+{
+    <NotH2 Class="pt-6 pl-6">@Name</NotH2>
+    <NotButtonCreate Class="mt-1 ml-6 mb-3" OnClick="_contestantNavigator.Create" />
+    <MudDivider />
+    @foreach (var item in Items)
+    {
+        <p class="mt-1 ml-6 mb-1">
+            @item.ToString() 
+            <div class="action-column">
+                <NotButtonUpdate Size="Size.Small" OnClick="() => this._contestantNavigator.Update(UpdateRoute, item)" />
+                <NotButtonDelete Size="Size.Small" OnClick="() => _parentBehind.Delete(item)" />
+            </div>
+        </p>
+        <MudDivider />
+    }
+}
+else
+{
+    <MudPaper Elevation="0">
+        <div class="pa-6">
+            <NotH2>@Name</NotH2>
+            <NotButtonCreate OnClick="_contestantNavigator.Create" Class="toolbar-button" />
+            <p>@NoItemsMessage</p>
+        </div>
+    </MudPaper>
+}
+</MudPaper>
+
+@code {
+    [Parameter]
+    public string Name { get; set; } = default!;
+    [Parameter]
+    public string NoItemsMessage { get; set; } = default!;
```

Comment:
This message is going to be the same for every entity. Why not simply construct it using the `Name` parameter and a string constant?

Reply:
Kept the parameter for custom messages but now setting the NoItemsMessage as a default.